DIST, VEL, ID, LABEL = 'dist', 'vel', 'id', 'label'
FLOW, COLS = 'flow', 'cols'
ACT = 'activation'
ACT0 = 'activation_0'
ACT1 = 'activation_1'
ACT2 = 'activation_2'

THR_ACT0 = 'thr_activation_0'
THR_ACT1 = 'thr_activation_1'
THR_ACT2 = 'thr_activation_2'
# LEFT, RIGHT, FORWARD, BACK = 'left', 'right', 'forward', 'back'